### ISSUE TITLE
phyloseq transform_sample_counts

### ISSUE THREAD

**Issue by vyom84** [joey711/phyloseq#624]
I am trying to calculate relative abundance in phyloseq using transform_sample_counts 

normx = transform_sample_counts(data, function(x) (x/sample_sums(x)))

This is the result I am getting using " otu_table(normx)[1:1, 1:2] "
            ABAFYQ    ABAKYD
OTU_1 0.0001313974      0.1883704

But When I do the same thing in vegan 
library(vegan)
normx<-decostand(data, “total”)*100

            ABAFYQ    ABAKYD
OTU_1 0.0001149954        0.1061447772 

The same results is obtained by doing 
normx=t(scale(t(data),center=FALSE,scale=rowSums(data)))

I don't know why the phyloseq relative abundance is not matching the vegan and R command. Please suggest, how to get the relative abundance I am getting from vegan in phyloseq. 

Thanks
Harinder Singh

**Comment by naarkhoo** [comment]
have you check if taxa_are_rows==true ? meaning, may be one is doing the operation on x the other on t(x);

**Comment by vyom84** [comment]
Yes taxa_are_rows is TRUE. We started using the new pipeline and the last OTU is unassigned, which is not consider in phyloseq. When I removed the unassigned OTU, it gives me the same normalisation as vegan does. 
I didn't check this error previously. 
Thanks for the help.
